integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -418,3 +418,10 @@ def test_arithmetic_reductions(data_gen):
                 'avg(a)'),
             conf = _no_nans_float_conf)
 
+@ignore_order
+@pytest.mark.parametrize('data_gen', all_gen, ids=idfn)
+@pytest.mark.parametrize('count_func', [f.count, f.countDistinct], ids=idfn)
```

Comment:
Using a function in here with idfn is producing different names each time the test runs and is causing the parallel runs of the tests to fail. If you drop the `ids=idfn` it fixes the problem